src/audio_worklet.js

```diff
@@ -38,60 +38,69 @@ function createWasmAudioWorkletProcessor(audioParams) {
     }
 
     process(inputList, outputList, parameters) {
+      // hardcoding this until the Web Audio 1.1 API spec makes it into browsers and we can get the AudioWorkletProcessor's context quantum
+      const quantumSize = 128;
```

Comment:
Reading the spec at https://webaudio.github.io/web-audio-api/ , I think we can (and should) already implement this, as it will be need to be implemented in a backwards compatible manner anyways.

I.e. something like `const quantumSize = context.renderQuantumSize || 128;`

The reason I say that we should implement this already now, because the data flow needs tending to, in order to get the `renderQuantumSize` posted into the worklet. So it would be good to implement it already pretending that the `renderQuamtumSize` parameter might exist in an implementation. Reading the spec, it seems fairly complete to assume that the quantum size will exist at some point.

Also, I wonder if we should expand the context creation function to allow taking in a quantum size?

Reply:
> I.e. something like `const quantumSize = context.renderQuantumSize || 128;`

Originally I did, but I chose not to in the end because it's `[SecureContext]` in the spec, and when it finally makes it to a browser I didn't want a exception. I thought of wrapping in a try/catch, but not knowing at all what the behaviour would be, I opted for the safer option.

> Also, I wonder if we should expand the context creation function to allow taking in a quantum size?

I also considered this, in preparation, but since I didn't know how it would be treated in reality I chose the safer option.

The rationale behind both decisions were: if we expose setting it, we need to be able to read it otherwise things will break, and if we read it things might also break. We're itching to see the larger buffer sizes at work, so it's definitely something I or @tklajnscek will revisit as soon as.